MaterialLib/Component/Dispersivity/Dispersivity.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   Dispersivity.h
+ */
+
+#pragma once
+
+#include "ProcessLib/Parameter/Parameter.h"
+
+namespace MaterialLib
+{
+namespace Component
+{
+class Dispersivity
+{
+public:
+    explicit Dispersivity(ProcessLib::Parameter<double> const& parameter)
```

Comment:
Why is there only one argument? I thought that dispersivity consists of longitudinal and transverse dispersivity.

Reply:
Exactly. The data structure has been rearranged to make it more clear.